### ISSUE TITLE
save and saveCanvas commands have stopped working

### ISSUE THREAD

**Issue by ricardocanada** [processing/p5.js#7372]
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [X] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### Steps:
1.
2.
3.

### Snippet:

```js

// Paste your code here :)

```

**Comment by ShivamH1** [comment]
From my end, both commands are running. 

![image](https://github.com/user-attachments/assets/e42e7da4-fcb9-43c1-b895-cb9d18ae8b1c)
![image](https://github.com/user-attachments/assets/d34dac51-bd4b-492e-9989-369e748e9541)

**Comment by ricardocanada** [comment]
Yes. The p5.js save command is working. The Processing save command opens a new tab in my browser, but with an error message, not an image.Richard BourneOn Nov 14, 2024, at 12:29 AM, ShivamH1 ***@***.***> wrote:﻿
From my end, both commands are running.
image.png (view on web)
image.png (view on web)
